app/shared/inputs/PhoneInput.js

```diff
@@ -31,6 +31,9 @@ function PhoneInput({
   hideIcon,
   shrink,
   required = false,
+  className,
+  placeholder,
+  useLabel = false,
```

Comment:
you just changed here every place that is using this component.
the label should be "phone" by default, so useLabel should be true by default

Reply:
@taylorgp this is still an issue